### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by DmytroBondariev**

### CODE REVIEW

File: cinema/views.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(data=serializer.data, status=status.HTTP_200_OK)
+    if request.method == "POST":
+        serializer = MovieSerializer(data=request.data)
+
+        if serializer.is_valid():
```
Comment (ArsenPidhoretskyi) [inline cinema/views.py:18]: Instead of if statement, use serializer.is_valid(raise_exception=True)

Comment (ArsenPidhoretskyi) [reply cinema/views.py:18]: Fix in all places

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+# Poetry local configuration file - https://python-poetry.org/docs/configuration/#local-configuration
+poetry.toml
+
+# ruff
+.ruff_cache/
+
+# LSP config files
+pyrightconfig.json
+
+# End of https://www.toptal.com/developers/gitignore/api/python,pycharm+all,django
```
Comment (vasylhnatiuk) [inline .gitignore:352]: add newline

File: cinema/migrations/0001_initial.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Movie',
```
Comment (vasylhnatiuk) [inline cinema/migrations/0001_initial.py:15]: change ' to "

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

**Review by vasylhnatiuk — APPROVED** [review]

**Pull request closed without merging**